esx_mechanicjob.sql

```diff
@@ -1,30 +1,21 @@
 USE `es_extended`;
 
 INSERT INTO `addon_account` (name, label, shared) VALUES
```

Comment:
This needs to be ALTER instead of INSERT because this value already exist in database

Reply:
Ohhh, this is the original file po from the master branch. Will change it nalang po